### DISCUSSION TITLE
Enable syntax highlight

### DISCUSSION THREAD

**Discussion by scripthunter7** [MajkiIT/polish-ads-filter#21239] [Ideas]
Hello!

GitHub now supports syntax highlighting of adblock filter lists, but this requires the presence of the adblock agent. Demo: https://github.com/ameshkov/VscodeAdblockSyntax/blob/master/test_rules.txt

If you want to enable it, just insert the appropriate agent at the beginning of your filters. For example `[Adblock Plus 2.0]` or `[uBlock Origin]` or `[AdGuard]`

**Comment by krystian3w** [comment]
I suppose this is the "rarest" readed section, main files has `[Adblock Plus 2.0]` or `[Adblock Plus 3.5]` but needed long time to load  highlighting (no actived asap after dom content loaded).

Supplement files may need add fallback in  `.gitattributes`

**Comment by hawkeye116477** [comment]
Only that file https://github.com/FiltersHeroes/PolishAnnoyanceFilters/blob/master/.gitattributes is needed :smile:.
